Modules/Devices/test/Devices.Application.Tests/Tests/Identities/Commands/RejectDeletionProcess/HandlerTests.cs

```diff
@@ -0,0 +1,129 @@
+﻿using Backbone.BuildingBlocks.Application.Abstractions.Exceptions;
+using Backbone.BuildingBlocks.Application.Abstractions.Infrastructure.UserContext;
+using Backbone.BuildingBlocks.Domain;
+using Backbone.Modules.Devices.Application.Identities.Commands.RejectDeletionProcess;
+using Backbone.Modules.Devices.Application.Infrastructure.Persistence.Repository;
+using Backbone.Modules.Devices.Domain.Entities.Identities;
+using Backbone.Tooling;
+using Backbone.UnitTestTools.Extensions;
+using FakeItEasy;
+using FluentAssertions;
+using Xunit;
+using static Backbone.UnitTestTools.Data.TestDataGenerator;
+
+namespace Backbone.Modules.Devices.Application.Tests.Tests.Identities.Commands.RejectDeletionProcess;
+public class HandlerTests
+{
+    [Fact]
+    public async void Happy_path()
+    {
+        // Arrange
+        var utcNow = SystemTime.UtcNow;
```

Comment:
I wanted to make this check but forgot about it, I inserted it now:
```csharp
    response.CreatedAt.Should().Be(utcNow);
```

If I don't setup the time in that fashion then the `CreatedAt` is a few milliseconds off.

Reply:
But why do you want to assert on something that's not the responsibility of the method you're testing?

Reply:
Well, I supposed the response is part of what handler does... Now that you asked that, I guess it kinda isn't... Not sure to be honest. Should I remove these:
```csharp
    response.Id.Should().Be(deletionProcess.Id);
    response.Status.Should().Be(DeletionProcessStatus.Rejected);
    response.CreatedAt.Should().Be(utcNow);
    response.RejectedByDevice.Should().Be(device.Id);
```

Reply:
It's not fixed, I didn't receive an answer to my last comment, but the conversation is still resolved.

Reply:
I don't understand what is happening as I didn't resolve it. It wouldn't even make sense to do so, since the last comment I made was a question for you. 

In any case, here is the screenshot:
![Screenshot 2024-02-29 102248](https://github.com/nmshd/backbone/assets/62119280/0e0008dd-94f4-44fb-a72c-16da43cea1d6)

Back to the issue. I realized the problem was that I was using the `CreatedAt` date whereas I should've been using `RejectedAt`. Fixed, but I will leave this unresolved for better visibility.